_posts/2015-12-13-weak-tests-hurt.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "Weak Tests Hurt"
+date:   2015-11-15 10:00:00
+categories: git development
+author: kent.safranski
+cover: /assets/images/covers/failed-test.jpg
```

Comment:
Yeah, I've got CDN/S3 in my notes moving forward, figure this works for now while we pick up steam.